styleguide/source/_patterns/02-molecules/forum-listing/forum-listing.twig

```diff
@@ -0,0 +1,41 @@
+{% if forumListing.showHeader %}
+<div class="ama__forum-table-wrap">
+    <table>
+        <thead>
+            <tr>
+                <th class="tl-forum-title">{{ forumListing.toplevelForum.title }}</th>
+                <th class="ama__column-centered forum-table-heading">Topics</th>
```

Comment:
When putting a string in a template file, it's best to set the translate filter. {{ 'topics'|t }}